examples/menu-button/menu-button-links.html

```diff
@@ -174,15 +174,15 @@ <h3 id="kbd2_label" class="widget">Menu</h3>
                 </ul>
               </td>
             </tr>
-            <tr>
+            <tr data-test-id="menu-home">
               <th><kbd>Home</kbd></th>
               <td>Moves focus to the first menu item.</td>
             </tr>
-            <tr>
+            <tr data-test-id="menu-end">
               <th><kbd>End</kbd></th>
               <td>Moves focus to the last menu item.</td>
             </tr>
-            <tr>
+            <tr data-test-id="menu-character">
```

Comment:
I see that we do not have tab and shift+tab documented for the menu, but we have scripting to close the menu without taking action and to go to next/prior element when the user tabs. After merge, I'll add that documentation with data-test-id attribs.

Reply:
ok!